pints/_mcmc/_mala.py

```diff
@@ -187,18 +183,15 @@ def ask(self):
             # Set as read-only
             self._proposed.setflags(write=False)
 
-        self._ready_for_tell = True
-
         # Return proposed point
         return self._proposed
 
     def tell(self, reply):
         """ See :meth:`pints.SingleChainMCMC.tell()`. """
 
         # Check if we had a proposal
-        if not self._ready_for_tell:
+        if self._proposed is None:
             raise RuntimeError('Tell called before proposal was set.')
-        self._ready_for_tell = False
 
```

Comment:
Sorry, probably not getting something but don't quite get why we'd remove these checks.

Reply:
It's more in line with what we have in the other MCMC methods, with the exception of the ones that do more complicated stuff in ask() than just set a proposal.

So no very strong reason! Just that it's possible to let users ask() as often as they like in this case (and always get the same result), and it lets us remove a variable...

We might want to disallow this everywhere instead though? In which case we'd have to update the documentation for SingleChainMCMC and MultiChainMCMC a bit to make this explicit...